terraform/transform_module_expansion.go

```diff
@@ -60,14 +83,13 @@ func (t *ModuleExpansionTransformer) transform(g *Graph, c *configs.Config, pare
 
 	// Add the root module node to provide a single exit point for the expanded
```

Comment:
You've changed the variable name, but I think this comment is out of date?

Reply:
"root" here refers here to _what_ it is, rather than its name. I'll add a little more context here.